database/repo.go

```diff
@@ -49,6 +49,7 @@ type Repo struct {
 	Link         sql.NullString `sql:"link"`
 	Clone        sql.NullString `sql:"clone"`
 	Branch       sql.NullString `sql:"branch"`
+	BuildLimit   sql.NullInt32  `sql:"build_limit"`
```

Comment:
Should this be `Limit` instead of `BuildLimit`?

I lean towards `BuildLimit` because it feels more descriptive on what the field is for.

However, it does diverge from the pattern established with `Counter` and `Timeout`.

If people like this new pattern, then I propose in a future PR, we update:

* `Counter` -> `BuildCounter`
* `Timeout` -> `BuildTimeout`

Reply:
I like `BuildLimit` and agree with your idea to implement that pattern for the other fields related to builds

Reply:
Any reason for choosing int32 over int64? If not, can we use int64 to be more consistent with the other items in the struct?

Reply:
@JordanSussman yeah the main reason was that `int32` is smaller than `int64` 👍 

In general, `BuildLimit` should contain a small value since we only allow a range of `1 - 30`.

Even if it was updated beyond `30`, I wouldn't expect the limit to exceed a value that `int32` couldn't support.

That being said, If we like changing it to `int64` for consistency than I can get behind that 😄

Reply:
@jbrockopp I would prefer `int64` for consistency.

Reply:
@JordanSussman this is complete 👍